pkgs/desktops/cinnamon/default.nix

```diff
@@ -1,5 +1,6 @@
 { pkgs, lib }:
 
 lib.makeScope pkgs.newScope (self: with self; {
-  xapps = callPackage ./xapps {};
+  cjs = callPackage ./cjs { };
```

Comment:
What about something like:

```suggestion
  # This is insecure but we still want Hydra to build it.
  unstafeCjsDoNotUse = callPackage ./cjs { };
  cjs = unstafeCjsDoNotUse // {
    meta = unstafeCjsDoNotUse.meta // {
      knownVulnerabilities = [
        "The SpiderMonkey runtime was extracted from Firefox 52, which EOL’d on September 5, 2018."
      ];
    };
  };

```

Reply:
The problem is that there are things that depend on cjs. As far as I understand it, we'd have to do this for every package that uses cjs as well.